bin/rtfm

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+require 'richmond'
+
+Richmond.generate ARGV[0]
```

Comment:
The neat thing about this is, you'd make it a command that anyone can readily use, versus having to open up irb.